net-im/bitlbee-facebook/bitlbee-facebook-1.1.2.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 2017 Gentoo Foundation
+# Copyright 2017-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
don't bump EAPI of the existing version, it would require revbump. it can be done with next upstream version bump naturally.

Reply:
I did not know that it should not be changed in the existing version, thanks for the info. Now I see, it is even written in Proxied Maintainer FAQ.